src/gui/dialogs/unit_recall.cpp

```diff
@@ -146,6 +146,14 @@ static std::string get_title_suffix(int side_num)
 	return msg.str();
 }
 
+void unit_recall:unit_recall_default_compare(unit &first, unit &second) {
+	if ( first.level() > second.level() ) return true;
```

Comment:
Wesnoth codebase practice is to omit space between keywords and opening parentheses, as well as within parentheses.
```
if(first.level() > second.level()) return true;
```